modules/sdk-coin-eth/src/eth.ts

```diff
@@ -429,6 +429,9 @@ export class Eth extends AbstractEthLikeNewCoins {
    * @returns {Promise<SignedTransaction>}
    */
   async signTransaction(params: SignTransactionOptions): Promise<SignedTransaction> {
+    if (params.isEvmBasedCrossChainRecovery) {
+      return super.signTransaction(params);
+    }
```

Comment:
please add a test to ensure, its calling the right function

Reply:
+1

Reply:
added a test